lib/executor/helpers.go

```diff
@@ -33,6 +33,32 @@ func getStagesUnscaledMaxTarget(unscaledStartValue int64, stages []Stage) int64
 	return max
 }
 
+func validateNumberOfVuShifts(startVus int64, stages []Stage) []error {
```

Comment:
Could we add documentation for this function please, especially regarding what the arguments to the function do, and what their constraints are 🙇🏻